package.json

```diff
@@ -39,7 +39,7 @@
   "scripts": {
     "build": "tsup",
     "build:watch": "tsup --watch",
-    "dev": "concurrently \"yarn build:watch\" \"vite --config examples/vite.config.ts --open\"",
+    "dev": "yarn build && concurrently --names tsup,vite \"yarn build:watch\" \"vite --config examples/vite.config.ts --open\"",
```

Comment:
what does `--names tsup,vite` do?

Reply:
Puts a prefix on the log output, so it's easier to see whether if an error is coming from the dev server (Vite) or the library builds (tsup). Not really necessary, but without it the prefixes would just be `[0]` and `[1]`.

![CleanShot 2025-04-17 at 09 06 23@2x](https://github.com/user-attachments/assets/ffdab513-25c5-4377-9c2b-ab7c12a48a42)